### PR TITLE
Add James and Jim to team

### PR DESCRIPTION
**Pull request by choldgraf**
This updates our Team page with a few changes after some recent personnel changes with @2i2c-org/2i2c-team: 

- Adds a new team called "Community Guidance and Partnerships"
- Lists @colliand and @jmunroe on this team
- Updates a few titles etc accordingly
- Removes @choldgraf from the engineering team and lists me as the Executive Director below
- Makes this a dedicated "Team" page

### CODE REVIEW

File: content/authors/james-munroe/_index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Community Guidance and Partnerships Team
+---
+
+TODO: @James can you suggest an edit here?
```
Comment (jmunroe) [inline content/authors/james-munroe/_index.md]: I'll probably revisit it in a few months, but for now how about this for some place holder text?

`James is Product and Community Lead for 2i2c. Coming from a background as an associate professor in the Department of Physics and Physical Oceanography at [Memorial University of Newfoundland](https://mun.ca), he is a strong advocate for enabling scientists and students to be efficient and effective in their computational workflows. Building on previous work in big data oceanography with links to the [Pangeo](https://pangeo.io/) project, [COSIMA](http://cosima.org.au/): Consortium for Ocean-Sea Ice Modelling in Australia, and [CIOOS](https://cioos.ca/): Canadian Integrated Ocean Observing System, James wants to bring the strength of the Jupyter ecosystem to users across a broad range of educational and research domains. `

**Review by jmunroe — COMMENTED** [review]
Looks to good to me. I've written a brief bio for myself. Happy to see this merged once @colliand has approved.

**Review by colliand — APPROVED** [review]
These changes look good to me.

File: content/authors/james-munroe/_index.md (hunk 1 above)
Comment (choldgraf) [inline content/authors/james-munroe/_index.md]:
```suggestion
James is Product and Community Lead for 2i2c. Coming from a background as an associate professor in the Department of Physics and Physical Oceanography at [Memorial University of Newfoundland](https://mun.ca), he is a strong advocate for enabling scientists and students to be efficient and effective in their computational workflows. Building on previous work in big data oceanography with links to the [Pangeo](https://pangeo.io/) project, [COSIMA](http://cosima.org.au/): Consortium for Ocean-Sea Ice Modelling in Australia, and [CIOOS](https://cioos.ca/): Canadian Integrated Ocean Observing System, James wants to bring the strength of the Jupyter ecosystem to users across a broad range of educational and research domains.
```

**Review by damianavila — APPROVED** [review]

**Comment by choldgraf** [comment]
ok let's merge this in! We can work on more "announcement" content later on but can consider this a soft-launch of @jmunroe and @colliand being "on the team" in an official sense :-)

**Pull request merged**